### PR TITLE
update maverick with zksync era data

### PR DESCRIPTION
**Pull request by gte620v**

### CODE REVIEW

**Comment by llamatester** [comment]
The maverick adapter exports: 
        
 
 _______________________________________
    Dexs for 19/4/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time: 20/2/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 865559757.884486
    Daily volume: 36452020.6866554
    Timestamp: 1681862400
    
    
    ERA 👇
    Backfill start time: 16/1/1970
    NO METHODOLOGY SPECIFIED
    Total volume: 532907.0397846256
    Daily volume: 211623.59432802876
    Timestamp: 1681862400
    
    
    
    
    Done in 4.39s.

**Comment by llamatester** [comment]
The maverick.ts adapter exports: 
        
 
 _______________________________________
    Fees for 19/4/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time: 20/2/2023
    Timestamp: 1681948798
    Daily fees: 3610.4390776868995
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Total fees: 92384.05165944947
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Daily user fees: 3610.4390776868995
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    Total user fees: 92384.05165944947
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    
    
    ERA 👇
    Backfill start time: 16/1/1970
    Timestamp: 1681948798
    Daily fees: 485.474563132966
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Total fees: 1108.8255513607212
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Daily user fees: 485.474563132966
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    Total user fees: 1108.8255513607212
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    
    
    
    
    Done in 4.18s.

File: dexs/maverick/index.ts [hunk 1]
```diff
@@ -9,6 +9,10 @@ const adapter: SimpleAdapter = {
       fetch: fetchVolume(CHAIN.ETHEREUM),
       start: async () => 1676851200,
     },
+    [CHAIN.ERA]: {
+      fetch: fetchVolume(CHAIN.ERA),
+      start: async () => 1337265,
```
Comment (dtmkeng) [inline dexs/maverick/index.ts]: i checked the subgraph there is a volume since 1681257600

File: fees/maverick.ts [hunk 2]
```diff
@@ -19,6 +19,13 @@ const adapter: SimpleAdapter = {
         methodology,
       },
     },
+    [CHAIN.ERA]: {
+      fetch: fetchFee(CHAIN.ERA),
+      start: async () => 1337265,
```
Comment (dtmkeng) [inline fees/maverick.ts]: I checked subgraph there is a volume since 1681257600

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The maverick adapter exports: 
        
 
 _______________________________________
    Dexs for 19/4/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time: 20/2/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 868636220.6219517
    Daily volume: 36452020.6866554
    Timestamp: 1681862400
    
    
    ERA 👇
    Backfill start time: 16/1/1970
    NO METHODOLOGY SPECIFIED
    Total volume: 579463.6220803909
    Daily volume: 211623.59432802876
    Timestamp: 1681862400
    
    
    
    
    Done in 9.37s.

**Comment by llamatester** [comment]
The maverick.ts adapter exports: 
        
 
 _______________________________________
    Fees for 19/4/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time: 20/2/2023
    Timestamp: 1681948798
    Daily fees: 3610.4390776868995
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Total fees: 92647.12149103938
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Daily user fees: 3610.4390776868995
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    Total user fees: 92647.12149103938
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    
    
    ERA 👇
    Backfill start time: 12/4/2023
    Timestamp: 1681948798
    Daily fees: 485.474563132966
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Total fees: 1209.0024790782231
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Daily user fees: 485.474563132966
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    Total user fees: 1209.0024790782231
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    
    
    
    
    Done in 5.75s.

**Comment by llamatester** [comment]
The maverick adapter exports: 
        
 
 _______________________________________
    Dexs for 19/4/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time: 20/2/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 868636220.6219517
    Daily volume: 36452020.6866554
    Timestamp: 1681862400
    
    
    ERA 👇
    Backfill start time: 12/4/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 579463.6220803909
    Daily volume: 211623.59432802876
    Timestamp: 1681862400
    
    
    
    
    Done in 4.31s.

**Comment by llamatester** [comment]
The maverick.ts adapter exports: 
        
 
 _______________________________________
    Fees for 19/4/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time: 20/2/2023
    Timestamp: 1681948798
    Daily fees: 3610.4390776868995
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Total fees: 92647.12149103938
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Daily user fees: 3610.4390776868995
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    Total user fees: 92647.12149103938
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    
    
    ERA 👇
    Backfill start time: 12/4/2023
    Timestamp: 1681948798
    Daily fees: 485.474563132966
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Total fees: 1209.0024790782231
    └─ Methodology: Fees generated on each swap at a rate set by the pool.
    Daily user fees: 485.474563132966
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    Total user fees: 1209.0024790782231
    └─ Methodology: LPs collect 100% of the fee generated in a pool
    
    
    
    
    Done in 4.03s.

File: dexs/maverick/index.ts (hunk 1 above)
Comment (gte620v) [reply dexs/maverick/index.ts]: fixed

File: fees/maverick.ts (hunk 2 above)
Comment (gte620v) [reply fees/maverick.ts]: fixed

**Comment by dtmkeng** [comment]
LG ty

**Pull request merged**